src/mongoose.ts

```diff
@@ -119,91 +125,81 @@ export function htMongooseFactory(mongoose: any) {
     TDocFactory extends DocumentFactory<any>,
     TInstance extends ReturnType<TDocFactory>,
     TUnsafeParam
```

Comment:
do we really need to specify the return type explicitly on the returned function?  There's a task for removing stuff like this wherever it's not needed.
- [ ] check if we need the `: HasSanitizeParams<TUnsafeParam, TSafeParam>`

Reply:
likewise everywhere

Reply:
you right we don't need to specify them
i remove them

Reply:
woohoo!